synapse/canvas/static/canvas-skills.css

```
@@ -0,0 +1,416 @@
+  flex-shrink: 0;
+}
+
+.skill-location-cell {
+  font-family: var(--font-mono);
+  font-size: var(--text-xs);
+  color: var(--color-text-muted);
+  line-height: 1.4;
+  max-width: 360px;
+  width: 360px;
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)